internal/plugin/service.go

```diff
@@ -43,9 +43,9 @@ func (s *serviceGenerator) generateConfigProvider(f *protogen.GeneratedFile) {
 	f.P("// that should be tested and how it's configured.")
 	f.P("type ", name, " interface {")
```

Comment:
This is technically a breaking change, right? But given that the interface feature is new, maybe it's OK... 🧐

What do you think?

Reply:
unfortunately yes, I discovered this issue when created one interface that embeds all the others, see this [example](https://github.com/einride/protoc-gen-go-aip-test/blob/97cfe0514b744656adc7b23a0234707ab208d27f/proto/gen/googleapis/aiplatform/apiv1/aiplatformpb/aiptest.pb.go), in that package two services has the same methods.

Reply:
IMO I think it's just breaking a feature that has been around for such a short time that a `fix()` commit is good enough!

Reply:
I agree with that, will change the scope of the commit message.